packages/clerk-js/src/ui/components/UserProfile/__tests__/UserProfile.test.tsx

```diff
@@ -49,10 +49,10 @@ describe('UserProfile', () => {
       expect(profileElements.some(el => el.tagName.toUpperCase() === 'BUTTON')).toBe(true);
       const securityElements = screen.getAllByText(/Security/i);
       expect(securityElements.some(el => el.tagName.toUpperCase() === 'BUTTON')).toBe(true);
-      const customElements = screen.getAllByText(/Custom1/i);
-      expect(customElements.some(el => el.tagName.toUpperCase() === 'BUTTON')).toBe(true);
-      const externalElements = screen.getAllByText(/ExternalLink/i);
-      expect(externalElements.some(el => el.tagName.toUpperCase() === 'BUTTON')).toBe(true);
```

Comment:
Update selectors using best practices https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-byrole-most-of-the-time